cove_ocds/test_input.py

```diff
@@ -0,0 +1,87 @@
+import os
```

Comment:
As you might guess, as a DRY fan, I don't particularly like that this has just been copy and pasted.